classes/blocks/class-carouselheader.php

```diff
@@ -37,7 +37,7 @@ public function __construct() {
 				'attributes'      => [
 					'block_style'       => [
 						'type'    => 'string',
-						'default' => 'zoom-and-slide-to-gray',
+						'default' => 'full-width-classic',
```

Comment:
Wouldn't it make sense to just completely remove this attribute if we cannot change it anyway?